documentation/asciidoc/accessories/camera/external_trigger.adoc

```diff
@@ -55,15 +44,35 @@ The low pulse width is equal to the shutter time, and the frequency of the PWM e
 
 NOTE: In this example, Pin 28 connects to the XTR touchpoint on the GS camera board.
 
-=== Operation
+=== Camera driver configuration
+
+This step is only necessary if you have more than one camera with XTR wired in parallel.
+
+As the superuser, edit `/boot/firmware/config.txt`. Change `camera_auto_detect=1` to `camera_auto_detect=0`.
```

Comment:
We generally don't use phrasing like "As the superuser..." - I think I'd probably just go with "Edit `/boot/firmware/config.txt` and change..."
You could link to https://www.raspberrypi.com/documentation/computers/config_txt.html#common-hardware-configuration-options if you really want to.

Reply:
I'll remove it. I suppose it was a sort of echo of a previous use of that [word](https://github.com/raspberrypi/documentation/blob/07dc7fa977899f14f9706916669acd96016c556d/documentation/asciidoc/accessories/camera/synchronous_cameras.adoc?plain=1#L22).